### PR TITLE
Fix depwarns for Julia 1.0

### PR DESCRIPTION
**Pull request by abhijithch**

### CODE REVIEW

File: src/Showoff.jl [hunk 1]
```diff
@@ -35,7 +35,8 @@ end
 # Floating-point
 
 function concrete_minimum(xs)
-    if done(xs, start(xs))
+    x = iterate(xs)
+    if x === nothing
```
Comment (Keno) [inline src/Showoff.jl]: `isempty` should work on 0.6 also, which may be slightly better?

Comment (abhijithch) [reply src/Showoff.jl]: Done.

File: REQUIRE [hunk 2]
```diff
@@ -1,2 +1,2 @@
-julia 0.6
+julia 0.7-alpha2
```
Comment (Keno) [inline REQUIRE]: If those are the only changes, I don't think this is necessary.

File: .travis.yml [hunk 3]
```diff
@@ -3,9 +3,9 @@ os:
...
 #  - osx
 julia:
-  - 0.6
+  - 0.7
   - nightly
 notifications:
   email: false
 after_success:
-  - julia -e 'cd(Pkg.dir("Showoff")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
+  - julia -e 'using Pkg; d(Pkg.dir("Showoff")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
```
Comment (Keno) [inline .travis.yml]: Errant `d` here.

File: .travis.yml [hunk 4]
```diff
@@ -3,9 +3,9 @@ os:
   - linux
 #  - osx
 julia:
-  - 0.6
+  - 0.7
```
Comment (Keno) [inline .travis.yml]: If we're keeping 0.6 below, keep it here.

File: REQUIRE (hunk 2 above)
Comment (abhijithch) [reply REQUIRE]: Sure

File: .travis.yml (hunk 3 above)
Comment (abhijithch) [reply .travis.yml]: Corrected

File: .travis.yml (hunk 4 above)
Comment (abhijithch) [reply .travis.yml]: ok

**Comment by daschw** [comment]
Can this be merged and released?
Plots fails because of the `done(..., start(...))` on 1.0

File: .travis.yml [hunk 5]
```diff
@@ -4,8 +4,9 @@ os:
 #  - osx
 julia:
   - 0.6
+  - 0.7
   - nightly
 notifications:
   email: false
 after_success:
-  - julia -e 'cd(Pkg.dir("Showoff")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
+  - julia -e 'using Pkg; cd(Pkg.dir("Showoff")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
```
Comment (mkborregaard) [inline .travis.yml:12]: is the `Pkg.dir` syntax not also deprecated in favor of `import Showoff; dirname(pathof(Showoff))`?

Comment (ararslan) [reply .travis.yml:12]: It is, but we're maintaining 0.6 support here I guess, so it'll have to stay as-is for the time being.

**Pull request merged**